### DISCUSSION TITLE
在使用自托管jitsi进行通话时,无法进入房间,页面一直加载中

### DISCUSSION THREAD

**Discussion by reboot523** [RocketChat/Rocket.Chat.ReactNative#6059] [General]
自托管jitsi使用标准80/443端口和ssl传输,并经过多端浏览器测试正常,但无法从rocketchat中使用通话和会议功能点击接听后页面无内容,页面中心有正在加载的圆圈.

**Comment by bezaleelcheung** [comment]
I met the same. Finding the answer ...

**Comment by Rohit3523** [comment]
Can you create an issue with detailed info instead of discussion please

**Reply by bezaleelcheung** [reply]
I submit a PR to resove this issue. #6507
